tests/drivers/test_forecast.py

```diff
@@ -0,0 +1,52 @@
+#pylint: disable=invalid-name, missing-module-docstring, missing-function-docstring
```

Comment:
I think it's good to remove the first line of the pylint disablers. Only the `missing-module-docstring` seems to apply and that may be one we want to fix by adding a docstring at the header of the file instead of ignoring.